ci/format.sh

```diff
@@ -18,7 +18,7 @@ unset CDPATH
 function follow_links() (
   cd -P "$(dirname -- "$1")"
   file="$PWD/$(basename -- "$1")"
-  while [[ -h "$file" ]]; do
+  while [[ -L "$file" ]]; do
```

Comment:
Consider adding a comment explaining what -L "$file" means.

Reply:
I just reverted it, that was a mistake.